app/person-escort-record/views/print-record.njk

```diff
@@ -123,6 +123,11 @@
                 reference: reference
               }) }}
             </span>
+            <div class="govuk-!-margin-top-2" data-tag-list-source="move-request">
```

Comment:
You can drop the `data` attribute. This was being used for an e2e check on the move detail page.